pkg/skaffold/build/local/jib_maven.go

```diff
@@ -30,8 +31,11 @@ import (
 )
 
 func (b *Builder) buildJibMavenToDocker(ctx context.Context, out io.Writer, workspace string, a *latest.JibMavenArtifact) (string, error) {
+	// If this is a multi-module project, we require `package` be bound to jib:dockerBuild
 	if a.Module != "" {
```

Comment:
Should we just move the check for `Module != ""` inside `verifyJibPackageGoal`?

Reply:
Currently we don't require jib to be bound to a package goal for the non-multimodule case.

Reply:
Oh I meant like just like at the top of `verifyJibPackageGoal`:

```go
if a.Module == "" {
  return nil // Does not need to verify if is not multi-module.
}
...
```

Reply:
But that seems at odds with the function name.  But I can declare `err` and combine into one line.

Reply:
Just a suggestion haha, not sure what the best practice here in Go is.

Reply:
I kinda of like it how it is, it makes it clear that multi module is handled differently.